test/uk/gov/hmrc/frontendtemplateprovider/FooterSpec.scala

```diff
@@ -36,17 +37,16 @@ class FooterSpec extends WordSpec with Matchers  with Results with WithFakeAppli
 
   implicit val hc = HeaderCarrier()
 
+  val fakeRequest = FakeRequest("GET", "/")
+
   override lazy val fakeApplication: Application = new GuiceApplicationBuilder()
-    .configure(Map("assets.url" -> "www.example.com/"))
-    .bindings(bindModules:_*)
+    .configure(Map("assets.url" -> "www.example.com/", "assets.version" -> "1"))
```

Comment:
Here it is overriding the values supplied by `application.conf`. `application.conf` is an implementation detail that should be ignored in test and is it would change to reflect local dev changes or environment changes and this is good practice to have this specified and tested in the application to validate assumption that the values are retrieved from the correct place.